core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java

```diff
@@ -635,6 +634,18 @@ private SqlNode getGroupBySqlNode(Builder builder, int key) {
     } */
   }
 
+  private boolean aliasNotRequiredInGroupBy(Builder builder, int key) {
```

Comment:
As this method returns a boolean value.
Could you please change name which start with either "is" or "does".
eg : isAliasNotRequiredInGroupBy

Reply:
Done